Core/Core/Planner/CalendarToDo/Model/API/APIPlannerNote.swift

```diff
@@ -22,15 +22,15 @@ import Foundation
 // Used only when creating or updating a PlannerNote (aka. Calendar ToDo),
 // not when fetching ToDos together with other plannables.
 public struct APIPlannerNote: Codable, Equatable {
```

Comment:
After testing, this API:
https://canvas.instructure.com/doc/api/planner.html#method.planner_notes.index

Seems to respond with list of PlannerNote objects with this specific format.

Note that documentation says otherwise about that, 
The main difference is the `description` property, this one is actually changed to be `details` exactly according this struct.